src/github/middleware.ts

```diff
@@ -128,7 +128,7 @@ export default (
 
 			if (await booleanFlag(BooleanFlags.MAINTENANCE_MODE, false, jiraHost)) {
 				context.log(`Maintenance mode ENABLED for jira host ${jiraHost} - Ignoring event of type ${webhookEvent}`);
-				return;
+				continue;
```

Comment:
We could have kept these 3 `returns`, but it's cleaner and more refactoring-safe if we never return from within the loop.

Reply:
Shame on me, I didnt' check for other returns!

Reply:
Let's extract the code within for loop to a separate function. I am still getting confused